cypress/integration/articleSpec.js

```diff
@@ -18,6 +18,13 @@ describe('News Article', () => {
 
     const anchorElement = getElement('header a');
     shouldContainText(anchorElement, 'BBC News');
+
+    anchorElement.focus();
+    shouldContainStyles(
```

Comment:
This test should be moved to their own explicit test. That should then be moved to its own header spec file so we can ensure seperation of concerns for testing against ACs.

Reply:
All other header tests should be moved to that spec file as well, but that is out of scope of this PR.

Reply:
@jamesbrumpton